apps/pie-storybook/stories/pie-list.stories.ts

```diff
@@ -1,19 +1,36 @@
 import { html } from 'lit';
 import { type Meta } from '@storybook/web-components';
 
-import '@justeattakeaway/pie-list';
 import { type ListProps } from '@justeattakeaway/pie-list';
+import '@justeattakeaway/pie-list';
+import '@justeattakeaway/pie-list/dist/pie-list-item';
 
 import { createStory } from '../utilities';
 
 type ListStoryMeta = Meta<ListProps>;
 
-const defaultArgs: ListProps = {};
+const defaultArgs: ListProps = {
+    variant: 'default',
+    hasDividers: false,
+    listType: 'unordered',
+};
 
 const listStoryMeta: ListStoryMeta = {
     title: 'Components/List',
     component: 'pie-list',
-    argTypes: {},
+    argTypes: {
+        variant: {
+            control: 'select',
+            options: ['default', 'compact'],
+        },
+        hasDividers: {
+            control: 'boolean',
+        },
+        listType: {
+            control: 'select',
+            options: ['unordered', 'ordered'],
+        },
```

Comment:
Let's use the exported defaults here instead of hardcoding them as hardcoding tends to get out of sync easily.

[example](https://github.com/justeattakeaway/pie/blob/main/apps/pie-storybook/stories/pie-card.stories.ts#L48-L51)
the changes would look like: 

```suggestion
        variant: {
            control: 'select',
            options: variants,
                defaultValue: {
                summary: defaultProps.variant,
            },
        }
```